services/dayService.js

```diff
@@ -43,10 +43,11 @@ export async function checkAndGetDate(dateToBeChecked) {
     const sanitizedDay = {
       id: day.id,
       date: day.date,
-      weight: day.weight,
-      height: day.height,
-      feastTime: day.feastTime,
+      weights: day.weights,
+      heights: day.heights,
+      feastTimes: day.feastTimes,
     };
     return sanitizedDay;
-  } else console.log("day not in db");
+  }
+  console.log("day not in db");
```

Comment:
you should use a console.error here or remove the log

Reply:
done